README.md

```diff
@@ -2,6 +2,12 @@
 
 Hubot script for writing birthday messages to users. It uses [Tenor](https://tenor.com) GIFs to make the messages more lively.
 
+## Features
+
+* Fetches a random GIF using the `simpsons-birthday` search query before writing a birthday message to a birthday boy/girl. The query (called a search term in the Tenor terminology) is configurable.
```

Comment:
Should be `simpsons+birthday`.

Reply:
`to a birthday boy/girl` --> `to a birthday person`

Reply:
`called a "search term"` (quotes)

Reply:
@ukhryab 

> Should be simpsons+birthday.

It used to be. Now it's `simpsons-birthday` (see https://github.com/tolstoyevsky/mmb/blob/hubot-rocketchat/hubot-rocketchat/docker-compose.yml#L20).

> to a birthday boy/girl --> to a birthday person

A birthday person is someone who is very fond of, so to say, birthday parties. It's not what I meant in the section=)

> called a "search term" (quotes)

I'll fix that.

Reply:
> It used to be. Now it's `simpsons-birthday`

Ok, then we should test it again. Cause last time I used it, there were no Simpsons on gifs.

> It's not what I meant in the section=)

So what did you mean by `birthday boy`? :) Isn't it the same that `birthday person`?
I meant that we can unite `boy/girl` in `person`.

Reply:
Ok, I've made a short research and found that you're right :)